concepticondata/conceptlists/Capell-1956-558.tsv

```diff
@@ -0,0 +1,559 @@
+Capell-1956-558-152	1398	DRY	152	dry	Capell1956[38]
+Capell-1956-558-153	1360	DUCK	153	duck (burdekin)	Capell1956[39]
+Capell-1956-558-154			154	dugong	Capell1956[39]
+Capell-1956-558-155	2683	EAGLE OR HAWK	155	eagle-hawk	Capell1956[39]
+Capell-1956-558-156	1247	EAR	156	ear	Capell1956[39]
+Capell-1956-558-157	1877	EAST	157	east	Capell1956[40]
+Capell-1956-558-158	1228	EARTH (SOIL)	158	earth	Capell1956[40]
+Capell-1956-558-159	1336	EAT	159	eat	Capell1956[40]
+Capell-1956-558-160	744	EGG	160	egg	Capell1956[40]
+Capell-1956-558-161	981	ELBOW	161	elbow	Capell1956[41]
+Capell-1956-558-162	1624	EMPTY	162	empty	Capell1956[41]
+Capell-1956-558-163	2817	EMU	163	emu	Capell1956[41]
+Capell-1956-558-164	1248	EYE	164	eye	Capell1956[41]
+Capell-1956-558-165	1173	EYEBROW	165	eyebrow	Capell1956[42]
+Capell-1956-558-166	1540	EYELASH	166	eyelash	Capell1956[42]
+Capell-1956-558-167	98	ALL	167	everything	Capell1956[42]
+Capell-1956-558-168	1560	FACE	168	face	Capell1956[42]
+Capell-1956-558-169	1280	FALL	169	fall	Capell1956[43]
+Capell-1956-558-170	323	FAT (ORGANIC SUBSTANCE)	170	fat (n.)	Capell1956[43]
+Capell-1956-558-171	1217	FATHER	171	father	Capell1956[43]
+Capell-1956-558-172	781	FEAR (FRIGHT)	172	fear	Capell1956[43]
+Capell-1956-558-173	1201	FEATHER	173	feather	Capell1956[44]
+Capell-1956-558-174	1551	FEMALE	174	female	Capell1956[44]
+Capell-1956-558-175	1242	FEW	175	few	Capell1956[44]
+Capell-1956-558-176	1423	FIGHT	176	fight	Capell1956[44]
+Capell-1956-558-177	2006	FILL	177	fill	Capell1956[45]
+Capell-1956-558-178	1303	FINGER	178	finger	Capell1956[45]
+Capell-1956-558-179	1766	FINISH	179	finish	Capell1956[45]
+Capell-1956-558-180	221	FIRE	180	fire	Capell1956[45]
+Capell-1956-558-181			181	firestick	Capell1956[46]
+Capell-1956-558-182	10	FIREWOOD	182	firewood	Capell1956[46]
+Capell-1956-558-183	227	FISH	183	fish	Capell1956[46]
+Capell-1956-558-184			184	fish-hawk	Capell1956[46]
+Capell-1956-558-185	19	FISHING LINE	185	fish-line	Capell1956[47]
+Capell-1956-558-186	388	FISHNET	186	fish-net	Capell1956[47]
+Capell-1956-558-187	1633	FLAT	187	flat	Capell1956[47]
+Capell-1956-558-188	2259	FLESH	188	flesh	Capell1956[47]
+Capell-1956-558-189	613	HIGH TIDE	189	flood	Capell1956[48]
+Capell-1956-558-190	1594	FLOUR	190	flour	Capell1956[48]
+Capell-1956-558-191	1504	FLY (INSECT)	191	fly (n.)	Capell1956[48]
+Capell-1956-558-192	1441	FLY (MOVE THROUGH AIR)	192	fly (v.)	Capell1956[48]
+Capell-1956-558-193	235	FLYING FOX	193	flying fox	Capell1956[49]
+Capell-1956-558-194			194	flying phalanger	Capell1956[49]
+Capell-1956-558-195	249	FOG	195	fog	Capell1956[49]
+Capell-1956-558-196	256	FOOD	196	food	Capell1956[49]
+Capell-1956-558-197	1301	FOOT	197	foot	Capell1956[50]
+Capell-1956-558-198	123	FOREHEAD	198	forehead	Capell1956[50]
+Capell-1956-558-199	1523	FORGET	199	forget	Capell1956[50]
+Capell-1956-558-200	503	FROG	200	frog	Capell1956[50]
+Capell-1956-558-201	1429	FULL	201	full (of water)	Capell1956[51]
+Capell-1956-558-202	1429	FULL	202	full (of food)	Capell1956[51]
```

Comment:
These two should maybe be unmapped, as they are quite specific about their meaning, but the concept is not